src/Teams/appsettings.Development.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Server=(local);Database=TeamCapacity;MultipleActiveResultSets=true;Trusted_Connection=True;"
+    "DefaultConnection": "Server=(local);Database=TeamCapacity;MultipleActiveResultSets=true;Trusted_Connection=False;"
```

Comment:
Why did u changed to false?

Reply:
Sorry, I was running this build on Linux, and for things to work on it the Trusted_Connections needed to be changed to false. Will change it back to true for github.